internal/render/helpers.go

```diff
@@ -95,6 +97,37 @@ func noMetric() metric {
 	return metric{cpu: NAValue, mem: NAValue, cpuLim: NAValue, memLim: NAValue}
 }
 
+func simpleMetric(cpu, mem resource.Quantity) metric {
+	return metric{
+		cpu: ToMillicore(cpu.MilliValue()),
+		mem: ToMi(client.ToMB(mem.Value())),
+	}
+}
+
+func percMetric(cpu, mem, rc, rm, lc, lm resource.Quantity, normalize bool) metric {
+	m := metric{
+		cpu:    IntToStr(client.ToPercentage(cpu.MilliValue(), rc.MilliValue())),
+		mem:    IntToStr(client.ToPercentage(client.ToMB(mem.Value()), client.ToMB(rm.Value()))),
+		cpuLim: IntToStr(client.ToPercentage(cpu.MilliValue(), lc.MilliValue())),
+		memLim: IntToStr(client.ToPercentage(client.ToMB(mem.Value()), client.ToMB(lm.Value()))),
+	}
+	if normalize {
```

Comment:
@sagor999 I was thinking adding a normalize method of the metric struct instead. So you should be able to move this logic into a normalize() method on metric and just call m.normalize(). Does this make sense? Tx!!

Reply:
@derailed problem with it, is that I need access to original rc value, to check if it was zero or not. since when it is converted into int, it becomes 0. which is valid value, in case if you pod for example consumes less than 1% of resources. So checking in metric if value is zero wouldn't be correct. 
I could save in metric struct original values... but that did not seem ideal as well.
```
if rc.IsZero() {
m.cpu = NAValue
}
```
This is the code that I am referring to. I need that original `rc` value. 
Unless I am misunderstanding you?

Reply:
@sagor999 Thank you I see what your are saying. Then perhaps we might be better off with something like a different Perc function ie render.toPercentStr() that will return NAValue if the divisor is zero or client.ToPercentage otherwise. That would be simpler IMHO. What do you think?

Reply:
@derailed yeah, it does look cleaner that way. Submitted changed. Let me know what you think.